source/php/Component/Fileinput/fileinput.blade.php

```diff
@@ -25,7 +43,7 @@ class="{{ $baseClass }}__input "
         @endicon
         <span>
             {{ $beforeLabel }}
-                {{ $label }}
+                {{ __('Select file') }}
```

Comment:
Will crash styleguide, trandslate function to avabile in styleguide since this is a wp-function.